src/main/java/pl/chojnacki/lesson10/task2/PeselValidator.java

```diff
@@ -0,0 +1,14 @@
+package pl.chojnacki.lesson10.task2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PeselValidator {
+    public static boolean validatePesel (String pesel) {
+
+        Pattern pattern = Pattern.compile("\\d{11}");
```

Comment:
Taka mała uwaga - tworzenie tego obiektu jest dość czasochłonne. AKtualnie tworzysz go za każdym razem jak wywołujesz metodę validatePesel. Aby tego uniknąć można to zapisać jako `private static final Pattern pattern = Pattern.compile([..])` i będzie to stworzone tylko raz.